src/Paket.Core/ProjectFile.fs

```diff
@@ -1337,6 +1338,18 @@ type ProjectFile with
 
     member this.FindReferencesFile() = this.FindCorrespondingFile Constants.ReferencesFile
 
+    member this.FindLocalizedLanguageNames() =
+        let pattern = @"Properties\\Resources.(?<language>\w+(-\w+)?).resx"
+        let includeattribute = "Include"
+        this.ProjectNode
+        |> getDescendants "EmbeddedResource"
+        |> List.filter (fun n -> n |> hasAttribute includeattribute)
```

Comment:
means `(fun n -> n |> hasAttribute includeattribute)` is the same as `(hasAttribute includeattribute)`